### DISCUSSION TITLE
Limiting Data in q-table

### DISCUSSION THREAD

**Discussion by CynthiaBlue** [quasarframework/quasar#12497] [CLI - Electron mode]
I have an array that has all my data in it. I want to only display certain data in the array in different q-tables.  Is this possible at the q-table level, or will I have to create a separate array for each q-table to be used as input?

Thanks. :)

**Comment by MilosPaunovic** [comment]
Create separate computed properties from your single source of truth and pass it down to your tables.

**Reply by CynthiaBlue** [reply]
ok... I can just reload the arrays with a function, too, instead of using a computed.  I actually have not used computed before so I don't know how, and not sure I want to go there. :p
